lib/src/formatter.dart

```diff
@@ -291,6 +266,57 @@ class SimpleFormatter implements ReportFormatter {
   }
 }
 
+writeBenchmarks(IOSink out, List<File> filesToLint, LinterOptions lintOptions) {
```

Comment:
Yep.  Consider this bit of code my last stand against gratuitous returns (in non API bits). 😸